webrender/src/render_backend.rs

```diff
@@ -214,6 +214,10 @@ impl RenderBackend {
                             }
                         }
                         ApiMsg::SetRootPipeline(pipeline_id) => {
+                            if scene.pipeline_map.get(&pipeline_id).is_none() {
```

Comment:
This doesn't look right to me. This is a setter function that checks for the value to already be there? When would it ever set the pipeline then?